pkg/usageStats/stats.go

```diff
@@ -523,8 +542,8 @@ func GetUsageStats(pastXhours uint64, granularity UsageStatsGranularity, orgid u
 // The data format has evolved over time:
 // - Initially, it was: bytes, eventCount, time
 // - Then, metrics were added: bytes, eventCount, metricCount, time
-// - Later, logsBytesCount and metricsBytesCount were added. However, the new format is backward compatible with the old formats.
-// The current format is: bytes, eventCount, metricCount, time, logsBytesCount, metricsBytesCount
+// - Later, logsBytesCount and metricsBytesCount were added: bytes, eventCount, metricCount, time, logsBytesCount, metricsBytesCount
+// The current format is bytes, eventCount, metricCount, time, logsBytesCount, metricsBytesCount, traceBytesAsCount, traceCount
 func parseStatsRecord(record []string) (ReadStats, error) {
```

Comment:
From the code in this function, it looks like it's still backwards compatible. Is that right? If so I think you should still mention somewhere in this comment that it's backwards compatible